master/internal/api_runs.go

```diff
@@ -310,7 +310,7 @@ func (a *apiServer) MoveRuns(
 		Column("r.id").
 		ColumnExpr("COALESCE((r.archived OR e.archived OR p.archived OR w.archived), FALSE) AS archived").
 		ColumnExpr("r.experiment_id as exp_id").
-		ColumnExpr("((SELECT COUNT(*) FROM runs r WHERE e.id = r.experiment_id) > 1) as is_multitrial").
+		ColumnExpr("(e.config->'searcher'->>'name' != 'single') as is_multitrial").
```

Comment:
nit (non-blocking): This is super nit-picky, and is more of a question about if we have a standard design practice in place for how we navigate nested jsonb fields in postgres. Both chaining `->->>` &  `#>>` will do the same thing, and IIRC '#>>` is just syntactic sugar.

Reply:
This is how it is done in other parts related to Run/Experiment Table, however there are a couple of instances where `#>>` is used. I think `->>` is more common though